app/models/check/base.rb

```diff
@@ -0,0 +1,25 @@
+module Check
+  class Base < OkComputer::Check
+    include HTTPRequestHelper
+
+    def report_failure(message)
+      mark_failure
+      mark_message message
+    end
+
+    def headers
+      { 'Content-Type': 'application/json' }
+    end
+
+    def simple_check_endpoint(url:, no_url_message:, fail_message:, success_message:)
+      if url
+        url = URI(url)
+        result = default_get_request_response(url: url, headers: headers)
+
+        result[:status] == 200 ? mark_message(success_message) : report_failure(fail_message)
```

Comment:
Magic number. Check https://gist.github.com/mlanett/a31c340b132ddefa9cca#gistcomment-2184336 and https://gist.github.com/mlanett/a31c340b132ddefa9cca#gistcomment-2988870.